python/ray/rllib/agents/ppo/appo.py

```diff
@@ -51,8 +59,65 @@
 # __sphinx_doc_end__
 # yapf: enable
 
+
+def make_aggregators_and_optimizer(workers, config):

```

Comment:
@michaelzhiluo wait do you even need this now that we've taken out modifications to the optimizer?